src_sycl/CMakeLists.txt

```diff
@@ -1,10 +1,13 @@
-find_package(GLM REQUIRED)
-find_package(GLFW REQUIRED)
-find_package(GLEW REQUIRED)
-set(OpenGL_GL_PREFERENCE GLVND)
+find_package(PkgConfig REQUIRED)
+
+pkg_check_modules(Glew REQUIRED IMPORTED_TARGET glew)
+
+find_package(glm REQUIRED)
+find_package(glfw3 REQUIRED)
 find_package(OpenGL REQUIRED)
 
-find_package(CUDA REQUIRED)
+set(MKL_DIR CACHE STRING "/opt/intel/oneapi/mkl/latest/lib/cmake/mkl")
```

Comment:
Maybe worth adding a sentence in the README about `MKL_DIR`?

Reply:
yeah that's probably wise

Reply:
I've added this in the last commit 52799714f7ea369b9d00a0db47f76b546d0b02a2